example/models_directory/unit_base_referable.json

```diff
@@ -0,0 +1,63 @@
+{
+    "flowchartId": "u123fndff333",
+    "modelPath": "pb/qm/dft/ksdft/hybrid",
+    "subtype": {
+        "name": "Hybrid functional",
+        "slug": "hybrid"
+    },
+    "name": "KS-DFT/B3LYP-D3(BJ)",
+    "functional": {
+        "name": "B3LYP-D3(BJ)",
+        "slug": "b3lyp-d3(bj)",
+        "components": [
```

Comment:
Same as above - we should probably store the components only when there is an adjustment to the fractions

Reply:
Or when there is a non-standard combination of exchange and correlation functionals.